crates/cli/src/commands/order/keys.rs

```diff
@@ -0,0 +1,236 @@
+use crate::execute::Execute;
+use crate::output::{output, SupportedOutputEncoding};
+use anyhow::{anyhow, Result};
+use clap::Parser;
+use rain_orderbook_common::dotrain_order::DotrainOrder;
+use std::fs::read_to_string;
+use std::path::PathBuf;
+
+#[derive(Debug, clap::ValueEnum, Clone, PartialEq)]
+pub enum KeyType {
+    Deployment,
+    Scenario,
+}
+
+#[derive(Parser, Clone)]
+pub struct Keys {
```

Comment:
can we rename this to `ListOrderFrontmatterKeys`

Reply:
Resolved